src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1002,14 +1004,157 @@ namespace
             asr::SunLightFactory().create(
                 light_name.c_str(),
                 asr::ParamArray()
-                  .insert("radiance_multiplier", intensity * asf::Pi<float>())
-                  .insert("turbidity", 1.0)
-                  .insert("environment_edf", sky_name)
-                  .insert("size_multiplier", size_mult)));
+                .insert("radiance_multiplier", intensity * asf::Pi<float>())
+                .insert("turbidity", 1.0)
+                .insert("environment_edf", sky_name)
+                .insert("size_multiplier", size_mult)));
         light->set_transform(transform);
         assembly.lights().insert(light);
     }
 
+    void add_analytical_light(
```

Comment:
This method doesn't appear to be specific to analytical lights, maybe `add_light_instance()` would be a clearer name?